prometheus_fzj_weather_exporter/main.py

```diff
@@ -27,7 +27,7 @@ def main():
         else:  # listen on all interfaces
```

Comment:
This is the behavior that changed upstream.

IIRC, the old behavior was to listen on all interfaces if it's not specified. Now it's 127.0.0.1.

I'm not certain though, so you should check into that.

Reply:
We talked and the changes around this will be addressed in another PR.